package.json

```
@@ -35,17 +35,16 @@
     "gatsby-plugin-postcss": "^2.0.0",
     "gatsby-plugin-react-helmet": "^3.0.1",
     "intersection-observer": "~0.5.0",
-    "ipfs": "~0.36.1",
+    "ipfs": "^0.37.0",
```

@alanshaw  CI fails with `Dependency ipfs had version ^0.37.0 - development (e.g. < 1.0.0) versions should start with a ~`
```suggestion
    "ipfs": "~0.37.0",
```